### DISCUSSION TITLE
File show

### DISCUSSION THREAD

**Discussion by rejetto** [rejetto/hfs#270] [Show and tell]
I'm working on a new feature to include in 0.46, it's called "file show", or just "show" for short.
Set of supported file formats can be expanded through plugins.

https://github.com/rejetto/hfs/assets/1367199/4e22eb13-ad1b-46aa-8305-c83986669e94

**Comment by GeorgeGeorgeGe** [comment]
Perfect. It would be great as well if you can display the file icon of each file type instead of the current generic one.
please let me know if the rename feature of file/folder would be available in the next version or not. 
Regards.

**Reply by rejetto** [reply]
please discuss this in dedicated place #250

**Comment by rejetto** [comment]
and maybe you want to show PDF as well?
go to admin > plugins > search
install [show-any](https://github.com/rejetto/show-any) plugin

**Reply by 6elkir** [reply]
I may have started a new branch taken to this (
https://github.com/rejetto/hfs/discussions/303

**Comment by anton-polo** [comment]
Hi @rejetto 
How to make the “show” mode open immediately when clicking on a file with an image ?  Not through the menu.

**Reply by rejetto** [reply]
the closest you can go is to hold SHIFT while clicking (as reported in the *readme* file).
To do it even without the shift, that's not an option, but it should be technically possible for a plugin using the `HFS.fileShow(entry)` api.
I just noticed this API was "announced" but not documented, but it will be soon.

**Comment by yb1536** [comment]
Hi @rejetto
I want to play a video taken with a 360 ° camera
Is it possible to move a 360 ° video using Three.js or Pannellum?

**Reply by rejetto** [reply]
the player is offered by the browser, so features and supported formats depend on the browser only. HFS does nothing about it.
